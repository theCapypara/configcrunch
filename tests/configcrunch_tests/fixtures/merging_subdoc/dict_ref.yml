base:
    str_field: test
    level_dict:
        no_merge:
          $ref: /level1
        merge:
          $ref: /level1
          more:
            merge_test: true
        no_ref:
          name: no-ref