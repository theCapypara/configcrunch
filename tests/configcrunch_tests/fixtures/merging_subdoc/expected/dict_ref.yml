base:
    str_field: test
    level_dict:
        no_merge:
          $name: no_merge
          name: level1
          more:
            merge_test: false
        merge:
          $name: merge
          name: level1
          more:
            merge_test: true
        no_ref:
          $name: no_ref
          name: no-ref
